lib/beacon/tailwind_compiler.ex

```diff
@@ -49,6 +49,10 @@ defmodule Beacon.RuntimeCSS.TailwindCompiler do
   end
 
   defp generate_tailwind_config_file(site, tmp_dir, content) do
+    # Copy tailwind base config so the generated file can use it
```

Comment:
If you take a look at the eex file, it now `require`s the base config. We need to copy it next to the generated config so it can find it.